doc/man3/OPENSSL_gmtime.pod

```diff
@@ -0,0 +1,62 @@
+=pod
+
+=head1 NAME
+
+OPENSSL_gmtime,
+OPENSSL_gmtime_adj,
+OPENSSL_gmtime_diff - platform-agnostic OpenSSL time routines
+
+=head1 SYNOPSIS
+
+ #include <openssl/crypto.h>
+
+ struct tm *OPENSSL_gmtime(const time_t *timer, struct tm *result);
+ int OPENSSL_gmtime_adj(struct tm *tm, int offset_day, long offset_sec);
+ int OPENSSL_gmtime_diff(int *pday, int *psec,
+                        const struct tm *from, const struct tm *to);
+
+=head1 DESCRIPTION
+
+OPENSSL_gmtime() returns the UTC time specified by B<timer> into the provided
+B<result> argument.
+
+OPENSSL_gmtime_adj() adds the offsets in B<offset_day> and B<offset_sec> to B<tm>.
+
+OPENSSL_gmtime_diff() calculates the difference between B<from> and B<to>.
+
+=head1 NOTES
+
+It is an error to call OPENSSL_gmtime() with B<result> equal to NULL. The
+contents of the time_t given by B<timer> are stored into the B<result>. Calling
+with B<timer> equal to NULL means use the current time.
+
+OPENSSL_gmtime_adj() converts B<tm> into a days and seconds value, adds the
+offsets, then converts back into a B<struct tm> specified by B<tm>. Leap seconds
+are not considered.
+
+OPENSSL_gmtime_diff() calculates the difference between the two B<struct tm>
+structures B<from> and B<to>. The difference in days is placed into B<pday>,
+the remaining seconds are placed to B<psec>. The value in B<psec> will be less
+than the number of seconds per day (3600). Leap seconds are not considered.
```

Comment:
86400 seconds in a day not 3600.

Reply:
Nah... I'm just going to restrict it to an hour... whoops.